CONTRIBUTING.md

```diff
@@ -0,0 +1,204 @@
+# Contributing
+
+We welcome new contributors to this community. If you are contributing to the
+CNCF and/or TAG-Security for the first time, it's okay if you feel overwhelmed.
+We, as a community, are here to help you with any problems you face. Open
+source is about collaboration and supporting each other.
+
+## Get Involved
+
+To become familiar with the team and understand how you will contribute, start
+by reading our [TAG Security charter].
+
+New members are advised to:
+
+- Join the [CNCF Slack], and introduce yourself on the the [#tag-security]
+  channel.
+- Review the following documents:
+  - [README.md]
```

Comment:
This link works for traversal in a local directory but is non-functional in the website.

Reply:
Oh this is frustrating. Do you see the different behavior between README and CODE-OF-CONDUCT?

The latter properly directs to `./CODE-OF-CONDUCT` while the former inexplicably goes to `contributing/README.md`.

Reply:
Troubleshooting... Confirmed they are both being parsed as local refs by `website/layouts/_default/_markup/render-link.html`

Reply:
Troubleshooting... In `render-link` README does not register as having a value for `$url.Fragment` while CODE-OF-CONDUCT does.

Reply:
Troubleshooting... The issue is because L7 of `renderlink` resets the value of $url, and if the file was a README it will be a null value at this point.

Reply:
Troubleshooting... L9-19 are responsible for removing the `contributing/` from the path. We need that cut in all situations, but for some reason we are hitting an edge case here.

Reply:
Since the problem was just with links to the top-level README, I added handling for that edge case specifically.